modules/developers-guide/pages/default-wallet.adoc

```diff
@@ -0,0 +1,192 @@
+
+[[wallet-create-wallets]]
+== Create a new cycles wallet
+
+Use the `wallet_create_wallet` method to create a new cycles wallet canister with an initial cycle balance and, optionally, with a specific principal as its controller.
+If you don't specify a controlling principal, the cycles wallet you use to create the new wallet will be the new wallet's controller.
+
+[[wallet-create-canister]]
+== Register a new canister identifier
+
+Use the `wallet_create_canister` method to register a new canister identifier on the {IC}. 
+This method creates a new "empty" canister placeholder with an initial cycle balance and, optionally, with a specific principal as its controller.
+After you have registered the canister identifier, you can install code for your application as a separate step.
+
+[[wallet-get-controllers]]
+== List the current controllers
+
+Use the `get_controllers` method to list the principals that have full control over a specified cycles wallet canister.
+
+For example:
+
+....
+dfx canister --network=alpha call f3yw6-7qaaa-aaaab-qaabq-cai get_controllers
+....
+
+The command returns a list of principals similar to the following:
+
+....
+(
+  vec {
+    principal "zen7w-sjxmx-jcslx-ey4hf-rfxdq-l4soz-7ie3o-hti3o-nyoma-nrkwa-cqe";
+    principal "vpqee-nujda-46rtu-4noo7-qnxmb-zqs7g-5gvqf-4gy7t-vuprx-u2urx-gqe";
+  },
+)
+....
+
+[[wallet-add-controller]]
+== Add a controller
+
+Use the `add_controllers` method to add a principal as a controller of the cycles wallet.
+
+For example:
+
+....
+dfx canister --network=alpha --no-wallet call f3yw6-7qaaa-aaaab-qaabq-cai add_controller '(principal "vpqee-nujda-46rtu-4noo7-qnxmb-zqs7g-5gvqf-4gy7t-vuprx-u2urx-gqe")'
+....
+
+In this example, the principal for the currently-active identity is a controller of the `f3yw6-7qaaa-aaaab-qaabq-cai` cycles wallet. The `--no-wallet` option enables the call to the add_controller method to use the principal for the currently-active identity (a controller) and not the cycles wallet (which is not a controller).
+
+To make the cycles wallet a controller of itself, you would run a command similar to the following:
+
+....
+dfx canister --network=alpha --no-wallet call f3yw6-7qaaa-aaaab-qaabq-cai add_controller '(principal "f3yw6-7qaaa-aaaab-qaabq-cai")'
+....
+
+[[wallet-remove-controller]]
+== Remove a controller
+
+Use the `remove_controller` method to remove a principal as a controller.
+
+For example:
+
+....
+dfx canister --network=alpha call f3yw6-7qaaa-aaaab-qaabq-cai remove_controller '(principal "zdm3q-uxmkz-lsghp-r737v-dgwav-rinn2-hs3zf-fnib3-2rylb-3kuek-hae")'
+....
+
+[[wallet-authorize]]
+== Authorize a custodian
+
+Use the `authorize` method to authorize a principal as a custodian of a cycles wallet.
+
+For example:
+
+....
+dfx canister --network=alpha call f3yw6-7qaaa-aaaab-qaabq-cai authorize '(principal "zdm3q-uxmkz-lsghp-r737v-dgwav-rinn2-hs3zf-fnib3-2rylb-3kuek-hae")'
```

Comment:
This format worked for me - i.e. no quotes for the wallet ID, however, the Quick Start guide uses quotes and this did not work for me — it threw an error (3) message.